gammapy/maps/tests/test_axes.py

```diff
@@ -757,3 +757,9 @@ def test_time_map_axis_format_plot_xaxis(time_intervals):
 
     ax1 = axis.format_plot_xaxis(ax=ax)
     assert ax1.xaxis.label.properties()["text"] == "Time [iso]"
+
+
+def test_single_valued_axis():
+    theta_values = np.array([0.5]) * u.deg
```

Comment:
Ah, here we are even mixing more things! You have a 1d array with one element which you pass to `Table`, which for astropy means "a column with a single row of scalar value".

This is the case that strictly speaking is not GADF compatible, since the format requires it to be an array (that could have only one element).

Could you also add the test for the compatible case of a single element? I.e. `theta_values = np.array([[0.5]]) * u.deg`?

Reply:
> which for astropy means "a column with a single row of scalar value".

I guess I hadn't realised this. In any case, I've expanded the test such that it also tests for the specs-compliant case. OK like so?

Reply:
Yes, perfect